lib/types/uuid.js

```diff
@@ -81,26 +97,16 @@ class Uuid {
      */
     toString() {
         // 32 hex representation of the Buffer
-        const hexValue = getHex(this);
-        return (
-            hexValue.substr(0, 8) +
-            "-" +
-            hexValue.substr(8, 4) +
-            "-" +
-            hexValue.substr(12, 4) +
-            "-" +
-            hexValue.substr(16, 4) +
-            "-" +
-            hexValue.substr(20, 12)
-        );
+        const hexValue = this.buffer.toString("hex");
```

Comment:
What does `"hex"` argument mean in `toString`?

Reply:
It is hex encoding (default is UTF-8), JS does not support named parameters to clearly specify what is passed to a function.